cmd/cmd_test.go

```diff
@@ -90,3 +91,30 @@ func (test *CommandTest) Teardown(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// terminalOverride lets us more easily redirect streams in the tests.
+type terminalOverride struct {
+	oldOut, oldErr, newOut, newErr io.Writer
+}
+
+// newTerminalOverride creates a new value to override the streams.
+func newTerminalOverride() terminalOverride {
+	return terminalOverride{
+		oldOut: Out,
+		oldErr: Err,
+		newOut: ioutil.Discard,
+		newErr: ioutil.Discard,
+	}
+}
```

Comment:
Nearly all of the tests discard the output, so that is the default behavior that I chose.